script.js

```diff
@@ -303,9 +304,39 @@ function validation() {
     document.getElementById('btn-form').type = 'submit';
     text.innerHTML = '';
   } else {
-    form.classList.remove('valid');
+    // form.classList.remove('valid');
     form.classList.add('invalid');
     document.getElementById('btn-form').type = 'button';
     text.innerHTML = '* Your email is invalid. (make sure your email is in lower cases)';
   }
-}
\ No newline at end of file
+}
+
+// Local Storage
+const form = document.getElementById('form');
+const nameField = document.getElementById('name');
+const emailField = document.getElementById('email');
+const messageField = document.getElementById('msg');
+
+const input = JSON.parse(localStorage.getItem('inputs'));
```

Comment:
think it would be clean to declare the variables at the top most part of the code, then use them inside whatever scope you choose.